packages/openid4vc/src/openid4vc-verifier/OpenId4VcSiopVerifierService.ts

```diff
@@ -424,20 +462,21 @@ export class OpenId4VcSiopVerifierService {
       .withRedirectUri(authorizationResponseUrl)
       .withIssuer(ResponseIss.SELF_ISSUED_V2)
       .withSupportedVersions([SupportedVersion.SIOPv2_D11, SupportedVersion.SIOPv2_D12_OID4VP_D18])
-      .withCustomResolver(getSphereonDidResolver(agentContext))
-      .withResponseMode(ResponseMode.POST)
+      .withResponseMode(ResponseMode.DIRECT_POST)
```

Comment:
Is this a breaking change?

Reply:
Will it still work with older versions (let's say we update Paradym Wallet first and Paradym later, or vice versa?)